### PR TITLE
LPS-125936 | master

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LPS-125936

This is only for master.

### CODE REVIEW

**Comment by Tim-Cao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase [hunk 1]
```diff
@@ -105,6 +105,16 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-96545. The edited default values are kept when cancel the Reset Values action in imported structure."
+	@ignore = "true"
+	@priority = "2"
+	test CancelResetImportedStructureDefaultValues {
+		property portal.upstream = "false";
+
+		// TODO LPS-123456 CancelResetImportedStructureDefaultValues pending implementation
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase:114]: Sorry I did a mistake in my example PR, this LPS should be the one related to the story we are automating. LPS-96545 in this case.

Same for all // TODO comments

**Comment by david-gutierrez-mesa** [comment]
Just started reviewing :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase (hunk 1 above)
Comment (david-gutierrez-mesa) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase:114]: Never mind. I fixed it locally :)

**Review by david-gutierrez-mesa — APPROVED** [review]

**Comment by david-gutierrez-mesa** [comment]
Thanks!! I'm creating a single PR with all these changes

**Comment by david-gutierrez-mesa** [comment]
See david-gutierrez-mesa#63

**Pull request closed without merging**